doc/Tests_XML_Definition_and_Sample.md

```diff
@@ -59,3 +60,159 @@ Example:
   </suite>
 </test_definition>
 ```
+
+# Tests.xml samples for supporting different test type
```

Comment:
The structure here is not correct.

Moreover, I prefer put together all the examples into one section under `tests.full.xml`.